config/default.json

```diff
@@ -37,5 +37,8 @@
 	"TSP_SCHOOL_SYNCER": {
 		"SCHOOL_LIMIT": 10,
 		"STUDENTS_TEACHERS_CLASSES_LIMIT": 150
+	},
+	"CTL_TOOLS": {
+		"EXTERNAL_TOOL_MAX_LOGO_SIZE_IN_BYTES": 300000
```

Comment:
INFO: We should discuss this with DevOps/SRE what size is reasonable

Reply:
I asked for it in rc. So i resolve it